src/ib_module.F

```diff
@@ -1634,7 +1634,7 @@ subroutine ib_lwr_flx(stag,ix,jy,kz,c1,c2,rrw,dumz,a,kbdy,vadvorder)
         i2=ni+1
       endif
 
-    !$acc parallel loop private(i,j,k,kval,wbar)
+    !$acc parallel loop default(present) private(i,j,k,kval,wbar)
```

Comment:
maybe add `gang vector` clause and `collapse` clause, and remove `private` clause?